### PR TITLE
Starting scylla with --kernel-page-cache from 4.5 version and above only

### PR DESCRIPTION
**Pull request by juliayakovlev**
This commit fixes the issue: https://github.com/scylladb/scylla-ccm/issues/318

Start Scylla node with --kernel-page-cache argument if the version is 4.5 or higher

Upgrade tests are passed: https://jenkins.scylladb.com/view/master/job/scylla-master/job/byo/job/dtest-byo/502/console

### CODE REVIEW

**Comment by fruch** [comment]
@juliayakovlev can you take a look at the unit tests running in github actions ? seem like this new code is failing the docker-ccm, and would need a different way to get the version for it.

**Comment by juliayakovlev** [comment]
> @juliayakovlev can you take a look at the unit tests running in github actions ? seem like this new code is failing the docker-ccm, and would need a different way to get the version for it.

@fruch I am checking it now. 
It failed on `common.isDse` function, that has been called from old  `ccmlib.node.Node.get_install_dir` without any parameter. For now it's not clear why it fails

**Review by fruch — APPROVED** [review]
LGTM, once it would pass the test, I'll merge it.

**Comment by juliayakovlev** [comment]
> LGTM, once it would pass the test, I'll merge it.

Upgrade and rolling tests are passed:
https://jenkins.scylladb.com/view/master/job/scylla-master/job/byo/job/dtest-byo/504/console

**Comment by fruch** [comment]
> > LGTM, once it would pass the test, I'll merge it.
> 
> @fruch
> Upgrade and rolling tests are passed:
> https://jenkins.scylladb.com/view/master/job/scylla-master/job/byo/job/dtest-byo/504/console

I'm fixing one more small thing cause the test in the PR were failing (they were using 666.development version)

**Comment by kbr-** [comment]
This broke my test workflow :(
getting these errors:
```
'/home/kbraun/dev/scylla/scylla-core-package/SCYLLA-VERSION-FILE' wasn't found
```
I guess I can reverse engineer the code and check what I need to put in `SCYLLA_VERSION_FILE` to make it work, but it would be nice if such breaking changes were documented.

**Pull request merged**
